src/CONST.ts

```diff
@@ -45,6 +45,10 @@ const CONST = {
     ARROW_HIDE_DELAY: 3000,
 
     API_ATTACHMENT_VALIDATIONS: {
+        // This is a list of file extensions that are not allowed to be uploaded to the server.
+        // TODO: This list should be updated to include all file types that are not allowed to be uploaded to the server.
+        UNALLOWED_EXTENSIONS: [''],
```

Comment:
I'll update this once I get input from @AndrewGable [here](https://github.com/Expensify/App/pull/23046/files#r1330313243)

Reply:
Looks like it was removed in https://github.com/Expensify/App/pull/21285 - So let's just remove this check!